shub/utils.py

```diff
@@ -830,6 +845,6 @@ def create_scrapinghub_yml_wizard(conf, target='default', image=None):
     if image or (image is None and _detect_custom_image_project()):
         repository = click.prompt(
             "Image repository (leave empty to use Scrapinghub's repository)",
-            default=True, show_default=False)
+            default=True, show_default=False, type=_BooleanOrStringParamType())
```

Comment:
This was failing since click 8.0.1, specifically since https://github.com/pallets/click/commit/ad52b19c84ab47cf6ec4df25b4e89ac939c3b253, which due to `default=True` was converting string inputs to `True` as well.